fabtests/functional/av_xfer.c

```diff
@@ -109,9 +109,9 @@ static int av_removal_test(void)
 		}
 	}
 
-	fprintf(stdout, "PASS\n");
 	(void) ft_sync();
 out:
+	fprintf(stdout, " %s\n",ret?"FAIL":"PASS");
```

Comment:
That's not what I meant -- I should have been more specific :) . No space needed inside the string. I would add space before 'ret', around '?' and around ':'.